app/controllers/api/v1/providers_controller.rb

```diff
@@ -31,6 +30,10 @@ def index
 
     private
 
+      def per_page
+        params[:per_page] || 100
```

Comment:
```suggestion
        (params[:per_page] || 100).to_i
```

https://github.com/DFE-Digital/teacher-training-api/blob/7226c70ec1e1aa34cd0a4294a18b30c86af77cc6/app/controllers/api/v1/courses_controller.rb#L41

Reply:
with your change `nil` would get converted to `0` the original would return `100`

Reply:
So is the others right then?

Reply:
I see, i've tweaked this and a few of the others